### PR TITLE
Added header support to task type bridge

### PR DESCRIPTION
**Pull request by george-dorin**
https://smartcontract-it.atlassian.net/browse/BCF-1966
https://github.com/smartcontractkit/chainlink/issues/8093

### CODE REVIEW

File: core/services/pipeline/task.bridge.go [hunk 1]
```diff
@@ -88,17 +89,23 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
...
 		includeInputAtKey StringParam
 		cacheTTL          Uint64Param
+		reqHeaders        StringSliceParam
 	)
 	err = multierr.Combine(
 		errors.Wrap(ResolveParam(&name, From(NonemptyString(t.Name))), "name"),
 		errors.Wrap(ResolveParam(&requestData, From(VarExpr(t.RequestData, vars), JSONWithVarExprs(t.RequestData, vars, false), nil)), "requestData"),
 		errors.Wrap(ResolveParam(&includeInputAtKey, From(t.IncludeInputAtKey)), "includeInputAtKey"),
 		errors.Wrap(ResolveParam(&cacheTTL, From(ValidDurationInSeconds(t.CacheTTL), t.config.BridgeCacheTTL().Seconds())), "cacheTTL"),
+		errors.Wrap(ResolveParam(&reqHeaders, From(NonemptyString(t.Headers), "[]")), "reqHeaders"),
```
Comment (pinebit) [inline core/services/pipeline/task.bridge.go:99]: Introducing a new feature requires to update documentation as well
(separate PR)
https://github.com/smartcontractkit/documentation

**Review by pinebit — COMMENTED** [review]
A few comments

File: core/services/pipeline/task.bridge.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (george-dorin) [reply core/services/pipeline/task.bridge.go:99]: The reason why it's named `reqHeaders` is to be consistent with the naming in HTTP task

Comment (george-dorin) [reply core/services/pipeline/task.bridge.go:99]: Understood, made PR here https://github.com/smartcontractkit/documentation/pull/1122

**Review by pinebit — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**